design/comments.go

```diff
@@ -100,12 +100,12 @@ var _ = a.Resource("comments", func() {
 
 	a.Action("show", func() {
 		a.Routing(
-			a.GET("/:id"),
+			a.GET("/:commentId"),
```

Comment:
I see it generates CommentID based on what is there now. Just wondering what it would generate if it was actually defined in the design as CommentID as well